### PR TITLE
#27 Fixed performance regression tests

### LINKED ISSUE #27: Performance regression tests fail
```
 │ Error: Your query returned no results. Please change your search criteria and try again.
│ 
│   with module.exasol_setup.module.exasol.data.aws_ami.exasol,
│   on .terraform/modules/exasol_setup.exasol/main.tf line 1, in data "aws_ami" "exasol":
│    1: data "aws_ami" "exasol" {
```

See https://github.com/exasol/azure-data-lake-storage-gen2-document-files-virtual-schema/actions/runs/4491744140/jobs/7900634950

### PR DESCRIPTION
**Pull request by kaklakariada**
Closes #27

### CODE REVIEW

File: cloudSetup/variables.tf [hunk 1]
```diff
@@ -6,7 +6,7 @@ variable "deputy" {
 
 # Use a short project tag. Long tags will cause the exasol cluster creation to fail (see https://github.com/exasol/cloudformation-aws-exasol/issues/3)
 variable "project" {
-  default = "VS-ADLS"
+  default = "VSADLS"
```
Comment (ckunki) [inline cloudSetup/variables.tf:9]: Can we use identical abbreviations in [`error_code_config.yaml`](https://github.com/exasol/azure-data-lake-storage-gen2-document-files-virtual-schema/blob/main/error_code_config.yml) and `cloudSetup/variables.tf`?

File: doc/changes/changes_1.3.1.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Virtual Schema for Document Data in Files on Azure Data Lake Storage Gen 2 1.3.1, released 2023-03-24
+
+Code name: Fix performance regression tests
+
+## Summary
+
+This release fixes execution of performance regression tests.
```
Comment (ckunki) [inline doc/changes/changes_1.3.1.md:7]:
```suggestion
This release fixes execution of performance regression tests and renames error codes from VSADLG2 to (?).
```

**Review by ckunki — CHANGES_REQUESTED** [review]

File: cloudSetup/variables.tf (hunk 1 above)
Comment (kaklakariada) [reply cloudSetup/variables.tf:9]: 👍

File: doc/changes/changes_1.3.1.md (hunk 2 above)
Comment (kaklakariada) [reply doc/changes/changes_1.3.1.md:7]: We will keep `VSADLG2`. The other code was wrong.

**Review by ckunki — APPROVED** [review]

**Pull request merged**